tests/test_info.py

```diff
@@ -195,3 +195,14 @@ def test_is_draft_pull_request(fake_github, title):
     # No matter what the title, a pr is Draft if it says it is.
     pr = fake_github.make_pull_request(title=title, draft=True)
     assert is_draft_pull_request(pr.as_json())
+
+def test_check_csv_users_only():
+    people = get_people_file()
+    user = 'Carlos-Muniz'
+    # This user exists in the yaml but not in the csv, and should not be in people
+    assert people.get(user) == None
```

Comment:
For whatever reason, it's idiomatic in Python to use `is None` rather than `== None`.

Reply:
I'm wondering if there's a way to assert that our test data is still what we think it is.  For example, your comment says you are in the yaml but not the csv.  Someone might unthinkingly edit the test data and break that assumption.  But this is probably beyond the scope of this pull request.

Reply:
Changed from == to is